src/components/Enum4Linux/Enum4Linux.tsx

```diff
@@ -46,109 +55,112 @@ const Enum4Linux = () => {
+    };
 
     return (
-        <form onSubmit={form.onSubmit((values) => onSubmit(values))}>
-            {LoadingOverlayAndCancelButton(loading, pid)}
-            <Stack>
-                {UserGuide(title, description_userguide)}
-                <TextInput
-                    label={"IP Address of Target"}
-                    placeholder={"Example: 192.168.1.200"}
-                    required
-                    {...form.getInputProps("ipAddress")}
-                />
-                <TextInput
-                    label={"Option"}
-                    placeholder={"Example: U"}
-                    required
-                    {...form.getInputProps("argumentMain")}
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
+            {!loadingModal && (
+                <InstallationModal
+                    isOpen={opened}
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
                 />
-                <TextInput label={"Parameters"} placeholder={"Example: o"} {...form.getInputProps("paramMain")} />
-                <TextInput label={"Additional Options"} {...form.getInputProps("argumentAlt")} />
-                <TextInput label={"Additional Options Parameters"} {...form.getInputProps("paramAlt")} />;
-                <Button type={"submit"}>Scan</Button>
-                {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
-                <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
-            </Stack>
-        </form>
+            )}
+            <form onSubmit={form.onSubmit(onSubmit)}>
+                {LoadingOverlayAndCancelButton(loading, pid)}
+                <Stack>
+                    <Switch
+                        size="md"
+                        label="Custom Configuration"
+                        checked={customMode}
+                        onChange={(e) => setCustomMode(e.currentTarget.checked)}
+                    />
+                    <TextInput label="IP Address of Target" required {...form.getInputProps("ipAddress")} />
+                    <TextInput label="Option" required {...form.getInputProps("argumentMain")} />
+                    <TextInput label="Parameters" {...form.getInputProps("paramMain")} />
+                    {customMode && (
+                        <>
+                            <TextInput label="Additional Options" {...form.getInputProps("argumentAlt")} />
+                            <TextInput label="Additional Parameters" {...form.getInputProps("paramAlt")} />
+                        </>
+                    )}
+                    {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
+                    <Button type="submit">Scan</Button>
```

Comment:
The convention is "Start <title of tool>"
```suggestion
                    <Button type="submit">Start { title }</Button>
```